src/Model/Gallery.php

```diff
@@ -162,13 +163,36 @@ public function getGalleryHasMedias()
     /**
      * {@inheritdoc}
      */
-    public function addGalleryHasMedias(GalleryHasMediaInterface $galleryHasMedia)
+    public function addGalleryHasMedia(GalleryHasMediaInterface $galleryHasMedia)
     {
         $galleryHasMedia->setGallery($this);
 
         $this->galleryHasMedias[] = $galleryHasMedia;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function removeGalleryHasMedia(GalleryHasMediaInterface $galleryHasMedia)
+    {
+        $this->galleryHasMedias->removeElement($galleryHasMedia);
+    }
+
+    /**
+     * @deprecated use addGalleryHasMedia method instead
+     * NEXT_MAJOR: remove this method with the next major release
+     *
+     * @param GalleryHasMediaInterface $galleryHasMedia
+     */
+    public function addGalleryHasMedias(GalleryHasMediaInterface $galleryHasMedia)
+    {
+        @trigger_error(
+            'The '.__METHOD__.' is deprecated and will be removed with next major release. Use `addGalleryHasMedia` method instead.',
```

Comment:
Line is too long, can you break it please, limit is 120 chars.

Reply:
fixed